src/redux/books/books.js

```diff
@@ -13,12 +16,40 @@ export const removeBook = (payload) => ({
   payload,
 });
 
+export const fetchBook = ((payload) => ({
+  type: FETCH_BOOK,
+  payload,
+}));
+
+export const addBookAPI = (payload) => async (dispatch) => {
+  const { id, title, category } = payload;
+  const newBook = { item_id: id, title, category };
+  await Axios.post('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/oA9H87RtWKrctkgy8p80/books', newBook);
+  dispatch(addBook(payload));
+};
+
+export const fetchBookAPI = () => async (dispatch) => {
+  const books = await Axios.get('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/oA9H87RtWKrctkgy8p80/books');
+  const mapBooks = Object.entries(books.data).map(([id, book]) => {
+    const { category, title } = book[0];
+    return { id, category, title };
+  });
+  dispatch(fetchBook(mapBooks));
+};
+
+export const removeBookAPI = (id) => async (dispatch) => {
```

Comment:
- [ ] The books removing feature seems to work well locally on the state, but not on the API.  If I add a book, then delete it, the book disappears.  However, if I reload the page the book reappears.

Kindly please correct the delete books feature so it's now possible to delete the correct books both locally and from the API.